lib/tools/doc/src/cover.xml

```diff
@@ -35,20 +35,21 @@
   <description>
     <p>The module <c>cover</c> provides a set of functions for coverage
       analysis of Erlang programs, counting how many times each
-      <em>executable line</em> of code is executed when a program is run.      <br></br>
-
+      <em>executable line</em> of code is executed when a program is run.
       An executable line contains an Erlang expression such as a matching
       or a function call. A blank line or a line containing a comment,
-      function head or pattern in a <c>case</c>- or <c>receive</c> statement
+      function head or pattern in a <c>case</c> or <c>receive</c> statement
       is not executable.</p>
     <p>Coverage analysis can be used to verify test cases, making sure all
       relevant code is covered, and may also be helpful when looking for
       bottlenecks in the code.</p>
     <p>Before any analysis can take place, the involved modules must be
-      <em>Cover compiled</em>. This means that some extra information is
+      <em>cover compiled</em>. This means that some extra information is
       added to the module before it is compiled into a binary which then
       is loaded. The source file of the module is not affected and no
-      <c>.beam</c> file is created.</p>
+      <c>.beam</c> file is created. If the runtime system supports coverage
+      natively, Cover will automatically use that functionality to lower the
+      execution overhead for cover-compiled code.</p>
```

Comment:
Maybe add a note that you can pre-compile modules compiling with `+line_coverage`?

Reply:
In a future pull request, I plan to make it possible for `cover` to take advantage of modules that are already loaded with `line_counters` enabled, but currently there is no advantage to pre-compiling modules with `line_coverage` when running `cover`.